app/controllers/concerns/content_items_concern.rb

```diff
@@ -0,0 +1,19 @@
+module ContentItemsConcern
```

Comment:
According to the card this isn't meant to be hardcoded and should apply to all attributes.. Also for clarity, the merits of this extract to me is of immediate benefit.. Also I prefer this to having a somewhat fat controller.